packages/dialect-react-ui/components/DisplayAddress/index.tsx

```diff
@@ -112,11 +105,7 @@ const SolanaName = ({
       </div>
     );
   }
-  return (
-    <div className="dt-flex dt-gap-1.5">
-      {`${formatSolanaDomain(displayName)}.sol ◎`}
-    </div>
-  );
+  return <div className="dt-truncate dt-mr-0.5">{`${displayName}.sol ◎`}</div>;
```

Comment:
why only mr? Previously it was `gap` (even thought I think since it had only 1 child it didn't properly work).

Reply:
Yes, I understand. But why the mr-0.5 then? Shouldn't we adjust the margin in the parent rather than here?